### DISCUSSION TITLE
forged and compiled client with own id and key download in browser causes warnings

### DISCUSSION THREAD

**Discussion by martinpetermal** [rustdesk/rustdesk#5512] [Q&A]
Hello.

I forged this project as mentioned here [()](https://rustdesk.com/docs/en/dev/build/all/).
Everything went well, except warnings in nearly all browser. 

why? and what to do to change it?
original rustdesk downloads are warning free.

kind regard - martin

**Comment by dinger1986** [comment]
Need to code signing certificate 👍

**Comment by martinpetermal** [comment]
thx, is there any tutorial, howto anywhere?

**Reply by dinger1986** [reply]
Not specific to rustdesk but loads out there
